### PR TITLE
Update createTelemetryItemExporter

### PR DESCRIPTION
**Pull request by heyams**

### CODE REVIEW

File: sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/utils/AzureMonitorHelper.java [hunk 1]
```diff
@@ -39,8 +41,7 @@ public static TelemetryItemExporter createTelemetryItemExporter(HttpPipeline htt
                         50, // default to 50MB
                         TempDirs.getSubDir(tempDir, "telemetry"),
                         telemetryPipeline,
-                        // TODO (trask) change this to statsbeatModule.getNonessentialStatsbeat()?
```
Comment (heyams) [inline sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/utils/AzureMonitorHelper.java:42]: exporter public API needs to be updated for customers to turn non-essential on. until then, default is OFF.

File: sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/utils/AzureMonitorHelper.java [hunk 2]
```diff
@@ -10,14 +10,16 @@
...
 import com.azure.monitor.opentelemetry.exporter.implementation.statsbeat.StatsbeatModule;
 import com.azure.monitor.opentelemetry.exporter.implementation.statsbeat.StatsbeatTelemetryPipelineListener;
+import reactor.util.annotation.NonNull;
 
 import java.io.File;
 
 public final class AzureMonitorHelper {
 
-    public static TelemetryItemExporter createTelemetryItemExporter(HttpPipeline httpPipeline, StatsbeatModule statsbeatModule, File tempDir) {
+    public static TelemetryItemExporter createTelemetryItemExporter(HttpPipeline httpPipeline, StatsbeatModule statsbeatModule, File tempDir, LocalStorageStats LocalStorageStats) {
```
Comment (trask) [inline sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/utils/AzureMonitorHelper.java:22]:
```suggestion
    public static TelemetryItemExporter createTelemetryItemExporter(HttpPipeline httpPipeline, StatsbeatModule statsbeatModule, File tempDir, LocalStorageStats localStorageStats) {
```

File: sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/utils/AzureMonitorHelper.java (hunk 1 above)
Comment (trask) [reply sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/utils/AzureMonitorHelper.java:42]: can you move this TODO up to the caller now? I suspect we will enable non-essential statsbeat by default to align with distro (and we can delete the TODO if we do not)

File: sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/utils/AzureMonitorHelper.java (hunk 2 above)
Comment (heyams) [reply sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/utils/AzureMonitorHelper.java:22]: it seems identical to me :)

Comment (trask) [reply sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/utils/AzureMonitorHelper.java:22]: Lowercase "L"

**Pull request merged**